charts/rancher-gke-operator/1.0.200/questions.yaml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.5.8-patch1
```

Comment:
Oh wait, I forgot GKE is getting remove entirely from 2.6. This PR likely isn't needed. It would only be needed if we keep it in system-charts. Sorry for double work.